src/main/java/codeu/controller/AdminServlet.java

```diff
@@ -63,5 +63,4 @@ public void doGet(HttpServletRequest request, HttpServletResponse response)
 
         request.getRequestDispatcher("/WEB-INF/view/admin.jsp").forward(request, response);
   }
-
```

Comment:
Delete whitespace?

Reply:
hmm i dont think i have any whitespace in my file rip

Reply:
oh wait the red means deleted! so that means i had deleted in a previous commit